### DISCUSSION TITLE
comando DOTNET RUN no terminal não funciona

### DISCUSSION THREAD

**Discussion by Anderson-Mendes-de-Almeida** [balta-io/forum#646] [Tira Dúvidas] [answered]
Boa noite Balta, cara tive problema com os arquivos Launch e Task, tive que configurar manual, pois deu problema e o VSCODE não os criava, agora ja consigo DEBUGAR com a IDE pelo botão, mas quando dou o comando DOTNET RUN no terminal da esse erro

Não foi possível localizar um projeto para executar. Verifique se existe um projeto em C:\dev ou passe o caminho para o projeto usando --project

è isso ai Balta!!!

**Comment by henriqueweege** [comment, marked as answer]
Oi, @Anderson-Mendes-de-Almeida ! Tudo certo? 
Só para ter certeza: quando você executa o dotnet run, está na pasta raiz do projeto?

Valeeeu!

**Comment by Anderson-Mendes-de-Almeida** [comment]
Cara eu to tendo serios problemas com os arquivos Launch e Task, meu VSCODE não cria ele automatico, e ta assim. tinha arrumado e agora voltou o problema, no google tem pouca informação, to te mandando os print.

![launch_error](https://github.com/user-attachments/assets/a6f51f00-ce97-491b-98c7-c1bc5ad48b8c)



![launch_error2](https://github.com/user-attachments/assets/5ff7759a-c7dc-4119-9d0f-523c77a5a64c)
![task_Error](https://github.com/user-attachments/assets/d232ced3-2e7c-4894-b165-5bfa1ee00386)

**Comment by thaiguer** [comment]
Nenhum caminho vai ter "\\" e "/".
Abre um powershell e digita o seguinte:
cd c:\
cd dev
dotnet new console -o ProjetoTeste
cd ProjetoTeste
dotnet run

![image](https://github.com/user-attachments/assets/54cda98e-db12-4f91-aa2a-f8b3b9dd9f5b)

Com isso, deveria aparecer hello world na tela.
Por fim, pode dar um: "code .", para abrir o projeto no VS code.
